packages/contracts/contracts/default-apps/ETHBalanceRefundApp.sol

```diff
@@ -13,27 +12,28 @@ contract ETHBalanceRefundApp {
     uint256 threshold;
   }
 
-  function resolve(bytes memory encodedState, Transfer.Terms memory terms)
-    public
+  function resolve(bytes calldata encodedState)
+    external
     view
-    returns (Transfer.Transaction memory)
+    returns (bytes memory)
   {
-    AppState memory state = abi.decode(encodedState, (AppState));
+    AppState memory appState = abi.decode(encodedState, (AppState));
 
-    uint256[] memory amounts = new uint256[](1);
-    amounts[0] = address(state.multisig).balance - state.threshold;
+    ETHInterpreter.ETHTransfer[] memory ret =
+      new ETHInterpreter.ETHTransfer[](1);
+    ret[0].amount = address(appState.multisig).balance - appState.threshold;
+    ret[0].to = appState.recipient;
```

Comment:
Is this `ETHInterpreter.ETHTransfer[]` object instantiable inline?

Reply:
I don't think so